### ISSUE TITLE
change project name

### ISSUE THREAD

**Issue by bdarcus** [bdarcus/citar#350]
Since merging support for CSL JSON, and as this project has become increasingly independent, I've been contemplating a name change.

In addition, only having a single recipe for the repo is not really ideal for installation ease (see #327), since different files have different dependencies.

So I suggest the following:

1. change the repo and all symbol and file names from "bibtex-actions" to something more general, but still distinctive
2. change the "oc-bibex-actions" prefix to an "org" suffix, and add a melpa recipe for it; can do the same for "latex"; this seems to make better sense even without 1.

I figure if I'm going to make these changes, best to do them at the same time, to minimize the inevitable disruption (one breaking change is better than two).

Name options, in rough order of my preference (though I'm not great with names):

1. **citar** (the spanish word for the similar english word "cite", but among english-oriented package names unique)
2. **excite** (suggested on the doom discord; I worry it's too general)

Both have the advantage of being one, not-too-long, word, with acii characters.

See #349 for the branch/PR, which currently is a start to implementing the 1 name; also, for melpa angle https://github.com/melpa/melpa/issues/7773.

![image](https://user-images.githubusercontent.com/1134/139543412-5bd93e0a-6d65-41ea-b85e-14c0fc438e1a.png)

Any input?

**Comment by bdarcus** [comment]
I submitted a name change PR for "citar". Will forego package-splitting, at least for now.

**Comment by salutis** [comment]
The annoying musical instrument everyone hates? Render me super-duper surprised. "Excite" would have been such an _amazing_ name. Good citations excite! What in the world...

**Comment by bdarcus** [comment]
> The annoying musical instrument everyone hates?

No; the Spanish word for "cite", and similar to the english word.

>  "Excite" would have been such an amazing name.

Well, you should have said the before I made the change ;-)

**Comment by salutis** [comment]
Oops! I did not comment because when I saw Excite, I thought: "Wow! No need to comment on the (planned) rename, for surely nobody will ever come up with anything better." (Not quite "woman mode"-level, but high up among the best names in Emacs.) In fact, I recompiled today and deleted `.emacs.d` (to test that my config works "from scratch"), and Emacs said `commandp, bibtex-actions-open`. I immediately thought: "OMG, OMG! Finally. Excite time!" Thrilled, I open `list-packages`, search for `excite`, and find ... nothing. Wait, what? Then, I came here...
